release-process.md

```diff
@@ -1,16 +1,36 @@
 # Release Process
-When preparing a standard release, do the following steps:
-
-- Branch the release branch off of the desired commit, naming it in the “release/v{major}.{minor}.{patch}” convention.
-- Bump the version number within the release branch, run npm shrinkwrap to update the shrinkwrap file also. Commit these two file changes to the repo.
-- Assist Application Support in verifying which Tracker stories are in the release, tagging said stories with a “{project}-v{major}.{minor}.{patch}” label.
-- Assist Application Support in creating the Release documentation, including the inclusion of all Pre and Post Deployment Actions in the documentation.
-- Deploy the release branch to the Staging environment, in coordination with the QA team. Follow the Release documentation exactly.
-- Once all regression testing is completed, at the request of Application Support, merge the release branch into “master”.
-    - Create a git tag in the format of “v{major}.{minor}.{patch}” on the merged master commit.
-    - Merge the updated master branch back into develop if present (rebase if possible).
-    - Clean up fully merged, release feature, epic, and release branches in git.
-- Review the final git state with Application Support.
-- Application Support will execute the production deployment using the release’s git tag.
-
-For hotfix releases, the same general process should be followed, except the branch name should be in the format of “hotfix/v{major}.{minor}.{patch}” (as stated above in the Git Branch Management section.
+
+## Continuous Delivery
+``` 
+Continuous Delivery is the ability to get changes of all types—including new features, configuration changes, bug fixes and experiments—into production, or into the hands of users, safely and quickly in a sustainable way.
+
+We achieve all this by ensuring our code is always in a deployable state, even in the face of developers making changes on a daily basis. We thus completely eliminate the integration, testing and hardening phases that traditionally followed “dev complete”, as well as code freezes. 
+```
+[Reference](https://continuousdelivery.com/)
+
+This is our goal when releasing software, however, we deal with real time operations of a business and sometimes need to schedule our releases to not be disruptive. 
+
+When permitted, all software should be released as soon as possible.
+
+## Immediate Releases
+When preparing a standard (immediate) release, do the following steps:
+
+- **Verify that the tests have passed in CircleCI, and the application runs as expected in the test space.**
+- Bump the version number within the release commit, commit this file change to the repo.
```

Comment:
Mention to bump the middle number for new features and last number for hotfixes

Reply:
just added below